FinniversKit/Sources/Recycling/ListViews/FavoriteFolders/FavoriteFoldersListView.swift

```diff
@@ -194,12 +194,12 @@ public class FavoriteFoldersListView: UIView {
             searchBar.resignFirstResponder()
         }
 
-        let numberOfItems = self.tableView(tableView, numberOfRowsInSection: Section.addButton.rawValue)
+        let numberOfItems = self.tableView(tableView, numberOfRowsInSection: Section.folders.rawValue)
```

Comment:
@osanoj do you know why it always been `addButton` here? It doesn't work... Could it be it actually never worked? :D